sc_rf_service/sc_rf_service.py

```diff
@@ -9,6 +9,12 @@
 
 import simulacrum
 
+from typing import List, Tuple
+import sys
+sys.path.insert(0, '../lcls-tools/lcls_tools/devices/')
+from scLinac import LINAC_TUPLES, Linac
+#from lcls_tools.devices.scLinac import Cavity, Cryomodule, LINAC_TUPLES, Linac
```

Comment:
Does this line not work?

Reply:
That line does not work on my mac. It only works in a pydm environment for me, but launching the simulacrum file in a pydm environment breaks in different ways. I think this is an issue of not having everything I need installed

Reply:
Oh yeah just do a `pip install --upgrade [path to your lcls-tools repo`

Reply:
`pip install --upgrade ~/SLAC/lcls-tools/` didn't seem to do the trick. I installed it and then killed and reran the sc_rf_service file, but I'm still getting an error saying ModuleNotFoundError: No module named lcls_tools. Am I missing something obvious?